package.json

```diff
@@ -3,30 +3,35 @@
   "private": true,
   "scripts": {
     "standardjs": "standard",
-    "stylelint": "stylelint **/*.scss",
-    "test": "jest"
+    "stylelint": "stylelint app/assets/stylesheets/**/*.scss",
+    "test": "jest",
+    "build": "esbuild app/javascript/*.* --bundle --sourcemap --outdir=app/assets/builds --public-path=assets --target=es6",
```

Comment:
> Make command descriptive, i.e yarn build:js

This command is mostly autogenerated (I added the `--target` flag).

It's added as part of the install step with [@rails/jsbundling-rails](https://github.com/rails/jsbundling-rails), and it [needs to be called `build` so that the relevant Rails tasks call it](https://github.com/rails/jsbundling-rails/blob/main/lib/tasks/jsbundling/build.rake).

---

> Can improve performance by ... Minifying compiled file

Sure, we can add a `--minify` flag.

---

> Can improve performance by ... Adding content hash to file name

[@rails/propshaft handles digest stamping for us](https://github.com/rails/propshaft#readme) already, if that's what you're referring to. 

I'm not familiar with cloud platform, so I'm not sure how or where we configure cache related headers etc. Are we using cloudfront?